ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js

```diff
@@ -27,13 +27,19 @@ const CustomInputField = (props) => {
       <TextField
         InputLabelProps={{ style : { pointerEvents : "auto", padding : "2px" } }}
         variant="outlined"
+        // style={{ width : "15rem" }}
```

Comment:
remove this. clean code, green code.

Reply:
Yes for sure, I'll update this, I'm with you `Clean code, green code` :)